src/common/server.c

```diff
@@ -572,6 +572,7 @@ ssl_do_connect (server * serv)
 			g_snprintf (buf, sizeof (buf), " * No Certificate");
 			EMIT_SIGNAL (XP_TE_SSLMESSAGE, serv->server_session, buf, NULL, NULL,
```

Comment:
Probably should remove this print then to avoid duplicate messages, also reform the above to fit in.